src/components/modal/v2/lib/getComputedVariables.js

```diff
@@ -0,0 +1,42 @@
+/**
+ *
+ * @param {*} offers Offers returned from the server.
+ * @returns Computed minAmount, maxAmount, belowThreshold, and aboveThreshold values.
+ *
+ * This function takes in the offers returned from the server and loops over each offer
+ * in order to find the absolute min and max amounts from the offers.
+ * From the offer with the lowest min amount, the belowThreshold error content is retrieved.
+ * Similarly with the offer with the highest max amount, the aboveThreshold error content is retrieved.
+ */
+const getComputedVariables = offers =>
```

Comment:
```suggestion
export const getComputedVariables = offers =>
```
How about instead we export from here so that we can destructure the function out from the lib we're already importing inside of Calculator.jsx and LongTerm/Content.jsx, rather than creating a new import statement just for this function. We can also just rename this file ```computeVariables```.